Report about failed schematron executions

juanluisrp opened the pull request:
If a schematron fails when it is running causing an exception now the user can know that an error has happened instead of see a successful 0/0 errors message.

Fixes #3594.

See warning:
![image](https://user-images.githubusercontent.com/826920/55010510-960bbc00-4fe4-11e9-9208-9dfdf706d34a.png)

archaeogeek commented:
Could the error message be changed to "Error happened... " rather than "Error happens..."? 

Also will that be clear enough to end users who are not admins? "calling the service" is not very descriptive

juanluisrp commented:
@archaeogeek changed the message to "Error happened...". About if it is clear enough, this message usually is caused by a bug in the schematron rules, so it is relevant to admins. What do you propose?

archaeogeek commented:
@juanluisrp maybe say: "An error occurred with this schematron, check the log files for more information"?

juanluisrp commented:
It is a good option for an expanded message, but it doesn't fit in the space available for the error count:
![image](https://user-images.githubusercontent.com/826920/55016819-d1f84e80-4fef-11e9-82bb-a051e62ed66e.png)

We need something even shorter than "Error happened while calling service".

juanluisrp commented:
Maybe something like "Server error" in the place of the error count, then "An error occurred with this schematron, check the log files for more information" below and then the actual error message?

MichelGabriel commented on web-ui/src/main/resources/catalog/components/edit/validationreport/partials/validationreport.html:
```diff
@@ -45,9 +45,11 @@
           <div class="col-md-9">
-            <h2 class="gn-schematron-title">{{(type.label || type.id) | translate}}</h2>
+            <h2 class="gn-schematron-title"><span data-ng-if="type.schematronVerificationError">
+              <i class="fa fa-exclamation-triangle text-danger"></i>&nbsp;</span>{{(type.label || type.id) | translate}}</h2>
```
I guess this is my only remark, I would rather see a class `fa-fw` than a `&nbsp;`

juanluisrp replied:
There are more `&nbsp;` in that template. Do I change them too?

juanluisrp commented:
@archaeogeek, I've changed the short message to "Server error"  and the expanded one, and moved it inside the error detail as "Please check server's log files for more information" below the actual message received from the server.

![image](https://user-images.githubusercontent.com/826920/55061915-2eea1800-5075-11e9-9656-87349b38bfe6.png)

archaeogeek commented:
Much better and easier for non-experts to understand :-)
